.travis.yml

```diff
@@ -5,6 +5,10 @@ sudo: true
 before_install:
   - scripts/install_toxiproxy.sh
 
+script:
+  - bundle exec rake
+  - bundle exec rubocop
```

Comment:
can you make this also part of the default rake task? otherwise people will run the tests locally and be surprised why they don't pass on CI